ocfweb/account/templates/account/vhost_mail/index.html

```diff
@@ -113,12 +117,66 @@
                         <span class="glyphicon glyphicon-plus" aria-hidden="true"></span>
                         Add address
                     </button>
+                    <h4>CSV import/export</h4>
+                    <form class="form-inline js-import-csv-form" enctype="multipart/form-data" action="{% url 'vhost_mail_csv_import' vhost.domain %}" method="post">
+                        <a class="btn btn-default" href="{% url 'vhost_mail_csv_export' vhost.domain %}">
+
+                            <span class="glyphicon glyphicon-download-alt" aria-hidden="true"></span>
+                            Download CSV
+                        </a>
+                        <div class="form-group">
+                            <label class="btn btn-default">
+                                <span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span>
+                                Upload CSV
+                                <input type="file" class="hidden js-import-csv" name="csv_file" accept="text/csv" />
+                            </label>
+                        </div>
+                        <a href="#csv_import_export">What's this?</a>
+                        {% csrf_token %}
+                    </form>
                 {% endwith %}
             {% endfor %}
             <hr />
             <small>
                 * Passwords can't be retrieved after being set, but you can always change them again.
             </small>
+            <div class="row">
+                <div class="col-md-8">
+                    <h4 id="csv_import_export">Importing and exporting CSVs</h4>
+                    <p>
+                        You can use CSV spreadsheets to manage your forwarding
+                        addresses. Clicking "Download CSV" will allow you to
+                        download all your forwarding addresses for a particular
+                        domain and the emails they forward to. Clicking "Upload
+                        CSV" will let you pick a CSV file to add new forwarding
+                        rules or update existing ones; no addresses will be
+                        deleted.
+                    </p>
+                    <p>
+                        The CSV format for downloading and uploading is UTF-8
```

Comment:
I feel like nobody will get the encoding part, I'd just mention the columns and what they contain